x-pack/plugins/cases/public/containers/api.test.tsx

```diff
@@ -125,6 +126,34 @@ describe('Case Configuration API', () => {
     });
   });
 
+  describe('resolveCase', () => {
+    const targetAliasId = '12345';
+    const basicResolveCase = {
+      outcome: 'aliasMatch',
+      case: basicCaseSnake,
+    };
+    const caseId = basicCase.id;
+
+    beforeEach(() => {
+      fetchMock.mockClear();
+      fetchMock.mockResolvedValue({ ...basicResolveCase, target_alias_id: targetAliasId });
+    });
+
+    test('check url, method, signal', async () => {
```

Comment:
nit: The test name strings should match what the test is doing so: maybe `test('resolveCase should be called with correct url, method, and signal')`